app.rb

```diff
@@ -2,13 +2,18 @@
 
 require 'json'
 require 'sinatra/base'
+require './lib/blockchain'
 require './lib/block'
 
 # Understands sending and receiving transaction hashes
 class BlockchainApp < Sinatra::Base
+  set :blockchain, Blockchain.new
+
   post '/blocks/create' do
     data = JSON.parse(request.body.read)['params']
-    Block.create(data['sender'], data['receiver'], data['value'])
+    settings.blockchain.create_transaction(data)
+    p settings.blockchain
+    halt 200
```

Comment:
What this is?

Reply:
L14 tells the blockchain instance to create a new transaction with the post parameters
L15 I will remove
L16 Closes the post request with a server response of 200